packages/components/src/DatePicker/DatePicker.mdx

```diff
@@ -46,6 +46,9 @@ import { DatePicker } from "@jobber/components/DatePicker";
 Use Datepicker when you are looking to provide the user with a highly visual
 interface for intuitive date selections.
 
+A classic use case for a standalone Datepicker would be as a means to navigate
+a more content-heavy calendar interface.
```

Comment:
"to navigate **to**"

Reply:
I don't think so? I'm thinking of something like this

![image](https://user-images.githubusercontent.com/39704901/144471608-11e068cd-386b-4e30-801f-d4263dfefdbc.png)

Or this
![image](https://user-images.githubusercontent.com/39704901/144471704-e9704d2e-3215-4807-a680-12d7926d0033.png)

Where the Datepicker is a tool for navigating the calendar, not really navigating _to_ the calendar?

Reply:
Maybe navigating _within_ the calendar is more clear?

Reply:
oh i see, yeah i definitely interpreted the original differently

Reply:
`within` makes sense

Reply:
```suggestion
within a larger calendar interface.
```